### PR TITLE
unit test coverage for utils, remove vestigial out-of-band context param

### PR DESCRIPTION
**Pull request by sklump**
... and `utils/temp.py`, no longer in play for revocation and now dead code
Signed-off-by: sklump <srklump@hotmail.com>

### CODE REVIEW

File: aries_cloudagent/utils/task_queue.py [hunk 1]
```diff
@@ -186,9 +186,9 @@ def __bool__(self) -> bool:
         """
         Support for the bool() builtin.
 
-        Otherwise, evaluates as false when there are no tasks.
+        Otherwise, evaluates as False when there are no tasks.
         """
-        return True
+        return bool(self.current_size)
```
Comment (andrewwhitehead) [inline aries_cloudagent/utils/task_queue.py:191]: Isn't that what the method is trying to avoid?

Comment (sklump) [reply aries_cloudagent/utils/task_queue.py:191]: * it strikes me as un-pythonic to have len() evaluate to zero and bool() come out True?
* this change matches the comments; the original does not, so something was wrong before?
* what is the point of a method that always returns True? Might as well not call it or, better, remove it?

Comment (andrewwhitehead) [reply aries_cloudagent/utils/task_queue.py:191]: The comment was a bit confusing, but the issue was that the entire task queue would evaluate to false if it didn't currently have any tasks running, which was confusing when checking for its existence elsewhere.

Comment (sklump) [reply aries_cloudagent/utils/task_queue.py:191]: OK, let me fix it and push the change tomorrow morning, thanks

**Pull request merged**